ros_ign_bridge/package.xml

```diff
@@ -21,8 +21,10 @@
   <!-- Default to citadel, support blueprint -->
   <depend condition="$IGNITION_VERSION == blueprint">ignition-msgs4</depend>
   <depend condition="$IGNITION_VERSION == blueprint">ignition-transport7</depend>
-  <depend condition="$IGNITION_VERSION != blueprint">ignition-msgs5</depend>
-  <depend condition="$IGNITION_VERSION != blueprint">ignition-transport8</depend>
+  <depend condition="$IGNITION_VERSION == citadel">ignition-msgs5</depend>
+  <depend condition="$IGNITION_VERSION == citadel">ignition-transport8</depend>
+  <depend condition="$IGNITION_VERSION == dome">ignition-msgs6</depend>
+  <depend condition="$IGNITION_VERSION == dome">ignition-transport9</depend>
```

Comment:
Previously, we would default to `citadel` if `IGNITION_VERSION` isn't set. I think it's important to have a default value, because users often overlook the need to set this variable.

Previously it wasn't possible to check if the variable wasn't set, that's why I checked against `!= blueprint`. But It should be possible since https://github.com/ros-infrastructure/rep/pull/288

Reply:
I'm not sure I did it right  here: feff3eca0a13a4fdd841291e48385a7534fb117a

Reply:
It looks like the `or` didn't work for colcon, `colcon graph` prints errors:

`ValueError: condition '$IGNITION_VERSION == dome or $IGNITION_VERSION == ''' failed to parse: Expected end of text, found 'o'`

I think we may need to have duplicate entries. These are the 2 commands that I use to check that the package files are working:

`colcon graph`

`rosdep check --from-paths src`

Reply:
I moved empty check to separate lines in 45356162ffdd39d874867bb9c3bda554633313b8.
I don't know how to use `colcon graph`, all I get when running it is `Project +` on the command line.

Reply:
Nice! I think that worked! I updated `ros_ign_gazebo_demos` in c4b4be7.

> I don't know how to use colcon graph, all I get when running it is Project + on the command line.


Yeah I can't parse the `+` s and `*`s either. What I do to test this is to symlink my blueprint / citadel / dome workspaces into my ROS workspace and then get a dot graph. Something like:

```
cd ~/ros_ws
ln -s ~/dome_ws/src src/dome_src
colcon graph --dot | grep "ros_ign.*->.*ignition"
```